### DISCUSSION TITLE
index.html not available in the repo

### DISCUSSION THREAD

**Discussion by graphicstone** [tradingview/lightweight-charts#899] [General]
I just downloaded the source code from [Lightweight Charts](https://github.com/tradingview/lightweight-charts), which was mentioned for the [Lightweight Charts](https://www.tradingview.com/lightweight-charts/) but I am not able to find the index.html file. There is only index.js there. Have I used an incorrect repo to download the source code or is there some other issue?

**Comment by timocov** [comment]
> but I am not able to find the index.html file

What do you expect to have in this file? Yes, we don't have it in the repo.

**Reply by graphicstone** [reply]
The index file for HTML, later I created that file on my own and it worked fine for me. One more thing can I disable the scroll and zoom feature in this? I want the graph to be fixed as it is, no movement at all.

**Reply by mariamlouis** [reply]
Please I need your help I have just download the source code How I work on it
